scrypto/src/resource/resource_flags.rs

```diff
@@ -1,23 +1,36 @@
+const RESOURCE_FLAGS_SHIFT: u8 = 0;
+const RESOURCE_FLAGS_MASK: u64 = 0xffff_ffff;
+
+macro_rules! resource_flags {
+    ( $f:expr ) => {
+        ($f as u64) << RESOURCE_FLAGS_SHIFT
+    }
+}
+
+pub fn resource_flags_are_valid(flags: u64) -> bool {
```

Comment:
perhaps should be inlined?

Reply:
Yes, agree. Simple inline hint would be good.